docs/release_notes.md

```diff
@@ -1,5 +1,36 @@
 # Release Notes
 
-## 1.0.0
+## Version 1.0.0
 
-To Be Defined
\ No newline at end of file
+### Added
+
+#### Core
+
+- Shift's workflow execution engine.
+- Shift's User Interface : An interface to interact with Shift and create workflows.
+- In-build catalogs:
+    - Constants catalog.
+    - Standard catalog.
+    - FileSystem catalog.
+    - Workflow catalog.
+    - CreativeSoftware catalog.
+- In-build plugins:
+    - Nodes Outliner plugin.
+    - Variables plugin.
+
+#### Integrations & Resources
+
+- Scripts to bind Shift interface to a DCC for:
+    - Maya: *mshift.py*.
+    - Houdini: *hshift.py*.
+    - Nuke: *nshift.py*.
+- USD resources:
+    - In-build catalogs: sUsd, sUsdTypes, sMayaUsd.
+    - In-build plugins: Usd Outliner, Usd Viewer.
+
+
+### Known Limitations
+
+#### Integrations & Resources
+
+- USD Viewer does not currently work in Houdini.
```

Comment:
After the test from @leegeertseninbibo maybe we have to remove this limitation

Reply:
Yes, after testing is confirmed that the latest distribution is working even in Houdini 20.5.332 with py310.